Description: Switch from \*(Ba to | and \*(Gt to > in man pages
 This follows recommendations from mdoc(7):

    As a special case, the predefined string \*(Ba is handled and rendered in
    the same way as a plain `|' character. Using this predefined string is not
    recommended in new manuals.

 Moreover, \*(Ba is unsupported by man-db and causes errors on systems which use
 it.

 Similarly, \*(Gt is both undefined in mdoc(7) and unsupported by man-db.
Author: Ryan Kavanagh <rak@debian.org>
Origin: vendor
Applied-Upstream: https://github.com/poolpOrg/OpenSMTPD/commit/9c8c5d09740c5c80a4960012449238efa09da8b0
 https://github.com/ryanakca/OpenSMTPD/commit/8cf5b0d311968591246c87f521c031c14c41b05f
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: opensmtpd/smtpd/aliases.5
===================================================================
--- opensmtpd.orig/smtpd/aliases.5	2014-02-01 14:33:51.764841442 -0500
+++ opensmtpd/smtpd/aliases.5	2014-02-01 14:33:51.764841442 -0500
@@ -60,7 +60,7 @@
 Append messages to
 .Ar file ,
 specified by its absolute pathname.
-.It \*(Ba Ns Ar command
+.It | Ns Ar command
 Pipe the message to
 .Ar command
 on its standard input.
Index: opensmtpd/smtpd/smtpd.conf.5
===================================================================
--- opensmtpd.orig/smtpd/smtpd.conf.5	2014-02-01 14:33:51.764841442 -0500
+++ opensmtpd/smtpd/smtpd.conf.5	2014-02-01 14:34:18.237653095 -0500
@@ -516,10 +516,10 @@
 .Pp
 Additional per-rule adjustments available:
 .Bl -tag -width Ds
-.It Ic expire Ar n Brq Ar s\*(Bam\*(Bah\*(Bad
+.It Ic expire Ar n Brq Ar s|m|h|d
 Specify how long a message that matched this rule can stay in the queue.
 .El
-.It Ic bounce-warn Ar n Bro Ar s\*(Bam\*(Bah\*(Bad Brc Bq , Ar ...
+.It Ic bounce-warn Ar n Bro Ar s|m|h|d Brc Bq , Ar ...
 Specify the delays for which temporary failure reports must be generated
 when messages are stuck in the queue.
 For example:
@@ -530,7 +530,7 @@
 will generate a failure report when an envelope is in the queue for more
 than one hour, six hours and two days.
 The default is 4h.
-.It Ic expire Ar n Brq Ar s\*(Bam\*(Bah\*(Bad
+.It Ic expire Ar n Brq Ar s|m|h|d
 Specify how long a message can stay in the queue.
 The default value is 4 days.
 For example:
@@ -889,7 +889,7 @@
 # touch /etc/mail/secrets
 # chmod 640 /etc/mail/secrets
 # chown root:_smtpd /etc/mail/secrets
-# echo "label username:password" \*(Gt /etc/mail/secrets
+# echo "label username:password" > /etc/mail/secrets
 # makemap /etc/mail/secrets
 .Ed
 .Pp
